project/src/main/kotlin/org/ionproject/core/common/Uri.kt

```diff
@@ -0,0 +1,22 @@
+package org.ionproject.core.common
+
+import org.springframework.web.util.UriTemplate
+
+object Uri {
+    private const val version = "/v0"
+
+    const val courses = "${version}/courses"
+    const val courseByAcr = "$courses/{course_id}"
+    const val klasses = "$courseByAcr/classes"
+    const val klassByTerm = "$klasses/{class_id}"
+    const val classSectionById = "$klassByTerm/{classSection_id}"
+    const val calendarByClass = "$klassByTerm/calendar"
+    const val calendarByClassSection = "$classSectionById/calendar"
```

Comment:
I realize using "calendarByClassSection = "$classSectionById/calendar" " and  " calendarByClass = "$klassByTerm/calendar" ", may be more efficient and clean. But its a lot harder to read for someone who is trying to deduce the URI's from this.